backend/src/main/java/edu/netcracker/backend/controller/ApproverCrudController.java

```diff
@@ -71,6 +71,14 @@ public BigInteger getRecordsCount() {
         for (User user : users) {
             userDTOS.add(UserDTO.from(user));
         }
+
```

Comment:
Please, delete this changes. It's not a cool!
`for (User user : users) {
            userDTOS.add(UserDTO.from(user));
}`

Reply:
You were wrong